Lib/configparser.py

```diff
@@ -724,16 +724,25 @@ def read(self, filenames, encoding=None):
         filename may also be given.
 
         Return list of successfully read files.
+
+        Any exception other than an OSError raised while
+        parsing the file is propogated to the caller.
         """
         if isinstance(filenames, (str, bytes, os.PathLike)):
             filenames = [filenames]
         encoding = io.text_encoding(encoding)
         read_ok = []
         for filename in filenames:
             try:
-                with open(filename, encoding=encoding) as fp:
+                fp = open(filename, encoding=encoding)
+            except (OSError, ValueError):
+                continue
+            try:
+                with fp:
                     self._read(fp, filename)
             except OSError:
+                # Do not silence ValueError during the parsing
+                # phase such as those due to a wrong encoding.
```

Comment:
This comment makes sense in light of this diff, but someone reading this comment in the future is likely to be confused, as this comment is above a 'continue' operation, so seems irrelevant. Since the tests protect against this condition, just omit the comment.